src/thunderbots/software/ai/hl/stp/tactic/move_tactic.cpp

```diff
@@ -2,38 +2,33 @@
 
 #include <algorithm>
 
-MoveTactic::MoveTactic(const Robot &robot) : Tactic(robot) {}
+MoveTactic::MoveTactic() : Tactic() {}
 
-std::unique_ptr<Intent> MoveTactic::updateStateAndGetNextIntent(const Robot &robot,
-                                                                Point destination,
-                                                                Angle final_orientation,
-                                                                double final_speed)
+void MoveTactic::updateParams(Point destination, Angle final_orientation,
+                              double final_speed)
 {
     // Update the parameters stored by this Tactic
-    this->robot             = robot;
     this->destination       = destination;
     this->final_orientation = final_orientation;
     this->final_speed       = final_speed;
-
-    return getNextIntent();
 }
 
-double MoveTactic::calculateRobotCost(const Robot &robot, const Field &field)
+double MoveTactic::calculateRobotCost(const Robot &robot, const World &world)
```

Comment:
Why this change?

Reply:
Future-proofing the code. I expect any goalie tactics to need to know about which robot is assigned to be the goalie according to the refbox, and since we're already using the `Team` I figured I'd just make it the `World`. This function will be called from the `PlayExecutor` level anyway which will already have a `World`.